README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
+- A number of smaller _project groups_.
```

Comment:
There's nothing to say a project group has to be smaller than the number of people on the dev team.
```suggestion
- A number of  _project groups_.
```

Reply:
I agree. If nobody objects, we should commit this change.

Reply:
It's your proposal, make the change.

Reply:
![](https://i.imgflip.com/2upjp5.jpg)